### ISSUE TITLE
Bugs encountered when running on KDE (Kubuntu, GNU/Linux)

### ISSUE THREAD

**Issue by NKID00** [RoderickQiu/wnr#47]
```
Test environment
OS: Kubuntu 21.04 x86_64
Kernel: 5.11.0-34-lowlatency
DE: Plasma 5.21.4
WM: KWin
```

- [ ] Tray icon is missing.
- [ ] Uncaught exception at start up. (Caused by the missing tray icon.)
  ```
  TypeError: Cannot read property 'setTitle' of null
    at IpcMainImpl.<anonymous> (wnr/main.js:2093:19)
    at IpcMainImpl.emit (events.js:315:20)
    at Object.<anonymous> (electron/js2c/browser_init.js:161:9692)
    at Object.emit (events.js:315:20)
    at Object.callbackTrampoline (internal/async_hooks.js:131:14)
  ```
- [ ] Tray tool tip is `electron`, not `wnr`. (After that the tray icon is added manually.)
- [ ] Left clicking the tray icon only has the same effect as a right-click. (After that the tray icon is added manually.)
- [ ] Focus is given to the menu icon (at the top-left corner of the main window) after making the main window shown up by the 'Show / Hide' button from the tray menu. (After that the tray icon is added manually.)
- [ ] Virtual desktops are not blocked by full-screen focus mode.

**Comment by RoderickQiu** [comment]
Thanks!
I assume the tray stuff be practical to be fixed, however, the last bug may need to make a complete new solution for it in the worst occasion (hopefully not, or a specified warning giver function might be a must...)
Also, if you are free, you may check about whether all the settings items really do their jobs fully on Linux, thank you dude.

**Comment by NKID00** [comment]
The `right-click` and `double-click` events of the class `Tray` are only available on macOS and Windows. This requires a change by Electron.

**Comment by RoderickQiu** [comment]
Yeah, you're right. Currently I can bear with the workaround of disable the former left-click behaviour of show/hide on Linux. Or if you feel like it, you can issue it to Electron.
And, is the transparent icon fixable or that you can go with it?

**Comment by RoderickQiu** [comment]
Anyway, I think I'm gotta release the new version. In case you have some new ideas, I can perform a hotfix to the release, which is to say, to replace the appimage of the release manually.

**Comment by NKID00** [comment]
This transparent icon issue does not affect normal use, only looks a bit weird though. I'll open another issue with more details later.
